README.md

```diff
@@ -109,8 +106,8 @@ Build a local archive by running the Maven package command.
 You can then invoke an example CLI application by adding the resulting jar file to your classpath. For example:
 
 ```text
-% java -cp target/random-cut-forest-1.0.jar com.amazon.randomcutforest.runner.AnomalyScoreRunner --help
-Usage: java -cp RandomCutForest-1.0-super.jar com.amazon.randomcutforest.runner.AnomalyScoreRunner [options] < input_file > output_file
+% java -cp target/random-cut-forest-1.0.jar com.amazon.com.amazon.randomcutforest.runner.AnomalyScoreRunner --help
```

Comment:
I'm not sure how this duplication happened, I'll remove it in the next revision.